doc/install.md

```diff
@@ -284,3 +288,13 @@ jupyter kernelspec list --json
 
 After that, you need to restart Jupyter, open your notebook, select the `pygmt` kernel
 and then import pygmt.
+
+
+### Not working transparency
+
+It is known that some combinations of GMT and ghostscript versions cause issues, especially
+regarding transparency:
+
+- ghostscript 9.53-9.56: work
+- ghostscript 10.00-10.01: doesn't work
+- ghostscript 10.02-10.03: work, but requires GMT<=6.4
```

Comment:
Instead of list gs versions that are working or not working, maybe we should just give some suggestions. For example, use gs 9.53-9.56 for GMT 6.3/6.4, use gs 10.03 or later for GMT 6.5.

Reply:
Started in commit https://github.com/GenericMappingTools/pygmt/pull/3249/commits/9da5d54093342151553e7a527adea197279f81b7